css/base.css

```diff
@@ -255,12 +255,19 @@ input[type='text'] {
 .tweet .text {
   clear: right;
   word-wrap: break-word;
+  white-space: pre-wrap;
```

Comment:
It was supposed to solve a problem that does not exist anymore. Multi-line retweets that span beyond the user's avatar were not aligning correctly. But I wasn't able to reproduce it. It is not needed anymore as it was solved somewhere else (altough I don't know where)